std/assembly/math.ts

```diff
@@ -946,10 +946,12 @@ export namespace NativeMath {
       p_h = reinterpret<f64>(reinterpret<u64>(p_h) & 0xFFFFFFFF00000000);
       p_l = v - (p_h - u);
       let z_h = cp_h * p_h;
-      let dp_l = select<f64>(dp_l1, 0.0, k);
+      // let dp_l = select<f64>(dp_l1, 0.0, k);
+      let dp_l: f64 = dp_l1 * <f64><bool>k;
```

Comment:
@dcodeIO Hmm, not sure `dp_l1 * <f64><bool>k` is right way. May be
```ts
dp_l1 * <f64>(k != 0)
```
Is better?

Reply:
I think the `: f64` annotation isn't necessary because `1.0` and `dp_l1` are already f64. `<bool>k` will compile to `i32.and(k, 1)` if I'm not mistaken, while `(k != 0)` is an `i32.ne(k, 0)`, hmm